example/print-manager-v2-example.ts

```diff
@@ -40,7 +42,7 @@ printProvider.init()
  *
  */
 const onChangePrintExtent = scale => {
-  const scaleSelect = document.querySelector('select#scale-select');
+  const scaleSelect: any = document.querySelector('select#scale-select');
```

Comment:
maybe you could use something like `HTMLElement` or `Element` here

Reply:
```suggestion
  const scaleSelect: HTMLElement = document.querySelector('select#scale-select');
```

or 

```suggestion
  const scaleSelect: HTMLElement | null = document.querySelector('select#scale-select');
```

Reply:
Obsolete, I'll change these back to `.js` due to them being used directly in the `.html` files.